content/02.introduction.md

```diff
@@ -121,7 +121,24 @@ The mechanism behind these insults has been suggested to be related to inflammat
 Individual outcomes have varied significantly, and reported hospitalization rates have been wide-ranging based on age and location, with various underlying health conditions being disproportionately reported [@doi:10.15585/mmwr.mm6915e3].
 Outcomes in patients that are hospitalized have generally been poor, with rates of admission to the intensive care unit (ICU) upwards of 15% in both Wuhan, China and Italy [@doi:10.1056/nejmoa2002032; @doi:10.1001/jama.2020.2648; @doi:10.1001/jama.2020.4031].
 
-Though older populations are generally considered the most vulnerable, pediatric infections are also a concern.
```

Comment:
There is not basis to say that - far more deaths in older cases and less in pediatric. You cannot compare them.

Reply:
I agree -- I put this paragraph in the wrong place.  I feel that pediatric infections and MIS/C need to be mentioned somewhere, but all of the other paragraphs in this subsection are about groups who are at a statistically higher risk, and this group is at a lower risk.  Maybe it would be better to move this paragraph into the subsection about clinical presentation.  Do you think so, @rando2?

Reply:
@dziakj1 that sounds reasonable to me! My understanding is that it's not unusual for pediatric cases to present somewhat differently than adult cases, so probably worth distinguishing! I'll see if I can get a pediatrician to read through it when we finish!

Reply:
Resolving this conversation because this topic is moved to #476